css/search.css

```diff
@@ -0,0 +1,340 @@
+.search-container--drinks {
+  display: flex;
+  flex-wrap: wrap;
+  cursor: default;
+  margin: 0 -15px;
+}
+
+.drink--wrapper {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 15px;
+}
+
+.drink {
+  display: flex;
+  align-items: center;
+}
+
+.drink-img {
+  width: 30%;
+  box-sizing: border-box;
+  border: solid;
+  border-color: #d4d4d4;
+  padding: 20px 30px;
+  border-width: 1px;
+  overflow: hidden;
+}
+
+.drink-img a {
+  width: 100%;
+  display: flex;
+  align-content: center;
+  justify-content: space-around;
+}
+
+.drink-img img {
+  max-width: 100%;
+  height: 250px;
+  transition: .5s;
+  pointer-events: none;
+}
+
+.drink:hover .drink-img img {
+  transform: scale(1.1);
+  transition: .5s;
+}
+
+.drink-info {
+  width: 70%;
+  padding-left: 30px;
+}
+
+.drink-name {
+  color: #000;
+  font-size: 1.5rem;
+  cursor: pointer;
+}
+
+.drink-name:hover {
+  color: #fca53c;
+}
+
+.container-litrage-and-prise {
+  display: flex;
+  flex-direction: column;
+}
+
+.drink-litrage {
+  color: #808080;
+  padding: 5px;
+  border: solid;
+  border-color: #fca53c;
+  border-width: 1px;
+  width: 40px;
+  text-align: center;
+  margin: 20px 0px 0px;
```

Comment:
zero doesn't have measure
zero is zero
doesn't matter is it % or pt
just write 0 and that's it